g2o/core/jacobian_workspace.cpp

```diff
@@ -55,9 +55,15 @@ bool JacobianWorkspace::allocate()
   }
   return true;
 }
-
-void JacobianWorkspace::updateSize(const HyperGraph::Edge* e_)
-{
+  
+  void JacobianWorkspace::updateSize(const HyperGraph::Edge* e_, bool reset)
+  {
+  if (reset == true)
```

Comment:
Can we drop the == true part here. Also at the occurrences below.

Reply:
Actually I don't think that reset feature is required - I forgot to remove it. Do you think the functionality is worth keeping? (I personally programme very defensively, but will change it if we keep.)

Reply:
I forgot this - I don't really use it at the moment. The use case was that the workspace could end up becoming very large. However, that doesn't happen in practice. Do you think the functionality is needed? If so, I'll change it. (I personally code very defensively.) If not, I'm happy to remove it.

Reply:
I've reformatted the code.